core/scripts/handle.py

```diff
@@ -0,0 +1,108 @@
+"""
+Handle handles creating and updating WCosa projects
+"""
+
+from shutil import copyfile
+from colorama import Fore
+from core.scripts.others.output import write, writeln
+from core.scripts.others import helper
+import core.scripts.templates.config as config
+import core.scripts.templates.cmake as cmake
+
+
+def create_wcosa(path, board, ide):
+    """Creates WCosa project from scratch"""
+
+    project_path = path
+
+    if path is None:
+        project_path = helper.get_working_directory()
+
+    if ide is None:
+        ide = ""
+    else:
+        ide = ide.strip(" ")
+
+    templates_path = helper.get_wcosa_path() + "/core/templates"
+    user_config_path = project_path + "/config.json"
+    internal_config_path = project_path + "/wcosa/internal-config.json"
+    general_cmake_path = project_path + "/wcosa/CMakeLists.txt"
+
+    write("Creating work environment - ", color=Fore.CYAN)
+
+    # create src, lib, and wcosa folders
+    helper.create_folder(project_path + "/src", True)
```

Comment:
This needs to check if `project_path` exists first, otherwise the following commands will fail.